spec/requests/users_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'If response status was correct.' do
+    it 'If response status was correct for users' do
+      get users_path
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'If response status was correct.' do
+    it 'If response status was correct for users ID' do
+      get user_path(1)
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'Render correct template' do
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template('index')
+    end
+
+    it 'renders the show template' do
+      get user_path(1)
+      expect(response).to render_template('show')
+    end
+  end
+
+  describe 'Response body includes correct placeholder text' do
+    it 'displays the correct placeholder text' do
+      get users_path
+      expect(response.body).to include('Here is the list of Posts')
+    end
+  end
+
+  describe 'Response body includes correct placeholder text' do
+    it 'displays the correct placeholder text' do
+      get user_path(1)
+      expect(response.body).to include('Here is the list of Posts')
+    end
+  end
```

Comment:
- [ ] Just like the first reviewer stated, these two tests fail when I try to run them. You can relate to my earlier [comment]() as the reason why the test in this file is failing. Please note that the tests are case-sensitive and you also need to crosscheck the texts used in the view files and ensure they are the same texts you are testing for. 👍 😎